NitroxServer/Communication/Packets/Processors/SetIntroCinematicModeProcessor.cs

```diff
@@ -26,6 +26,7 @@ public override void Process(SetIntroCinematicMode packet, Player player)
         packet.PartnerId = null; // Resetting incoming packets just to be safe we don't relay any PartnerId. Server has only authority.
         player.PlayerContext.IntroCinematicMode = packet.Mode;
         playerManager.SendPacketToOtherPlayers(packet, player);
+        Log.Debug($"Set IntroCinematicMode to {packet.Mode} for {player.PlayerContext.PlayerName}");
```

Comment:
Do we want to keep this log or was this just used for testing?

Reply:
I want to keep this log. It's very helpful when something breaks to know the state the server had of the modes